contracts/sol6/reserves/bridgeReserve/uniswap/KyberUniswapReserve.sol

```diff
@@ -0,0 +1,339 @@
+pragma solidity 0.6.6;
+
+import "../../../IKyberReserve.sol";
+import "../../../IERC20.sol";
+import "../../../utils/WithdrawableNoModifiers.sol";
+import "../../../utils/Utils5.sol";
+import "../../../utils/zeppelin/SafeMath.sol";
+import "../../../utils/zeppelin/SafeERC20.sol";
+
+library UniswapV2Library {
+    using SafeMath for uint256;
```

Comment:
its best if we can have this library in a separate file. so we can easily update / upgrade if required.